pallets/auctions/src/lib.rs

```diff
@@ -44,16 +42,115 @@ const BID_ADD_BLOCKS: u32 = 10;
 /// Minimal auction duration
 const MIN_AUCTION_DUR: u32 = 10;
```

Comment:
such configuration parameters would be probably better to have as pallet constants